docs/source/fundamentals.md

```diff
@@ -1,5 +1,7 @@
-## Fundamental concepts
+# Fundamental concepts
 In this section we will present some of the main concepts behind SimPhoNy.
+
+## General notions
 ### Operability
```

Comment:
I see that according to some definitions of the word "operability", that word might not be the choice for the title. From Wikipedia:
> Operability is the ability to keep a piece of equipment, a system or a whole industrial installation in a safe and reliable functioning condition, according to pre-defined operational requirements.

Suggestion: "Level of interoperability"